lib/reaction_for_like_site_setting_enum.rb

```diff
@@ -3,19 +3,15 @@
 require_dependency 'enum_site_setting'
 
 class ReactionForLikeSiteSettingEnum < EnumSiteSetting
-  HEART ||= 'heart'
-
   def self.valid_value?(val)
     values.any? { |v| v[:value] == val }
   end
 
   def self.values
-    @values ||= begin
-      reactions = SiteSetting.discourse_reactions_enabled_reactions.split('|').map do |reaction|
+    @values = begin
```

Comment:
removed `||` because values of valid_reactions can change and that causes the issue.